doc/tutorials/upgrading_configuration.rst

```diff
@@ -0,0 +1,70 @@
+.. _upgrading_configuration:
+
+Upgrading ACRN Configurations to Recent Releases
+################################################
+
+The configurations files, introduced in :ref:`acrn_config_data`, are refined
+every release for richer features, clearer organizations and more user friendly
+representations. Due to the strict validation ACRN adopts, configuration files
+for a former release may not work for a latter if they are not upgraded. This
+tutorial introduces the steps to upgrade those files.
+
+Board XML
+*********
+
+A board XML file lists the characteristics of a board and is generated by the
+board inspector. Each release of ACRN adds some more information into that file
+in order to enable new features. Thus, always regenerate the board XML using the
+board inspector of the new release when you upgrade.
```

Comment:
Sure would be nice if the board inspector wrote the ACRN release version in the board XML file so we could detect a potential problem early (using an old board XML).

Reply:
@NanlinXie @gvancuts Do we want this for v3.0 or v3.1?

Reply:
In any case, that's not a problem with this doc PR :)